services/transactionpool/committed_pool.go

```diff
@@ -11,12 +12,27 @@ import (
 type committedTxPool struct {
 	transactions map[string]*committedTransaction
 	lock         *sync.RWMutex
+
+	metrics *committedPoolMetrics
+}
+
+type committedPoolMetrics struct {
+	transactionCountGauge *metric.Gauge
+	poolSizeInBytesGauge  *metric.Gauge // TODO use this metric
 }
 
-func NewCommittedPool() *committedTxPool {
+func newCommittedPoolMetrics(factory metric.Factory) *committedPoolMetrics {
+	return &committedPoolMetrics{
+		transactionCountGauge: factory.NewGauge("TransactionPool.CommittedPool.TransactionCount"),
```

Comment:
what does this Gauge represent? is it number of transactions seen since startup, or current size of pool?

Reply:
Tx currently in the pool.